python/lsst/analysis/tools/actions/plot/scatterplotWithTwoHists.py

```diff
@@ -672,17 +672,17 @@ def _makeTopHistogram(
     ) -> None:
         # Top histogram
         topHist = figure.add_subplot(gs[0, :-1], sharex=ax)
+        x_min, x_max = ax.get_xlim()
+        bins = np.linspace(x_min, x_max, 100)
 
-        if "any" in self.plotTypes:
-            x_all = data[f"x{self._datatypes['any'].suffix_xy}"]
-            keys_notall = [x for x in self.plotTypes if x != "any"]
+        if "all" in self.plotTypes:
+            x_all = f"x{self._datatypes['all'].suffix_xy}"
+            keys_notall = [x for x in self.plotTypes if x != "all"]
+            topHist.hist(x_all, bins=bins, color="grey", alpha=0.3, log=True, label=f"All ({len(x_all)})")
```

Comment:
Why are you still defining an `x_all` below the `else`?  Ditto for `y_all` below.  Actually, you can probably move this setting of `keys_notall` above the `if` and just get rid of the `else`?  Same below, again.

Reply:
The all should probably be any which brings it in line with the key used in the datatype option. It is for if people don't care if it is a star or a galaxy and just want to plot everything.